frontend/components/TableContainer/DataTable/StatusCell/StatusCell.tsx

```diff
@@ -12,13 +17,32 @@ const generateClassTag = (rawValue: string): string => {
   return rawValue.replace(" ", "-").toLowerCase();
 };
 
-const StatusCell = ({ value }: IStatusCellProps): JSX.Element => {
+const StatusCell = ({ value, tooltip }: IStatusCellProps): JSX.Element => {
   const statusClassName = classnames(
     "data-table__status",
     `data-table__status--${generateClassTag(value)}`
   );
-
-  return <span className={statusClassName}>{value}</span>;
+  console.log("tooltip text:", tooltip?.tooltipText);
+  const cellContent = tooltip ? (
+    <>
+      <div data-tip data-for={tooltip.rowId}>
+        {value}
+      </div>
+      <ReactTooltip
+        className="status-tooltip"
+        place="top"
+        type="dark"
+        effect="solid"
+        id={`${tooltip.rowId}`}
+        backgroundColor="#3e4771"
+      >
+        {tooltip.tooltipText}
+      </ReactTooltip>
+    </>
+  ) : (
+    <>{value}</>
```

Comment:
This component and the one in `HostSummary.tsx` look nearly identical. If they both display a status with tooltip text can this be pulled out and reused in both places?

Reply:
@ghernandez345 yes! I did just that in [this draft PR](https://github.com/fleetdm/fleet/pull/8698#pullrequestreview-1179271850) if you want to take a look at that